apps/api/infra/api.ts

```diff
@@ -161,6 +161,18 @@ export const serviceSetup = (services: {
       ISLYKILL_SERVICE_PASSPHRASE: '/k8s/api/ISLYKILL_SERVICE_PASSPHRASE',
       ISLYKILL_SERVICE_BASEPATH: '/k8s/api/ISLYKILL_SERVICE_BASEPATH',
       IDENTITY_SERVER_CLIENT_SECRET: '/k8s/api/IDENTITY_SERVER_CLIENT_SECRET',
+      FINANCIAL_STATEMENTS_INAO_BASE_PATH:
+        '/k8s/api/FINANCIAL_STATEMENTS_INAO_BASE_PATH',
```

Comment:
Basepath might be an env variable if it is open and well known (they are not trying to hide it in some way)